src/pgsql/bin/postgres/entrypoint.sh

```diff
@@ -38,6 +38,9 @@ chown -R postgres $PGDATA && chmod -R 0700 $PGDATA
 
 /usr/local/bin/cluster/repmgr/configure.sh
 
+echo '>>> Creating symlinks'
```

Comment:
I think it should be done only one after initial cluster setup. not every time on start.